MSAL/test/unit/utils/MSALTestConstants.h

```diff
@@ -39,4 +39,4 @@
 #define UNIT_TEST_DEFAULT_REDIRECT_URI      UNIT_TEST_DEFAULT_REDIRECT_SCHEME"://auth"
 
 #define UT_SLICE_PARAMS_DICT DEFAULT_SLICE_PARAMS
```

Comment:
so this now becomes nothing. which means tests like MSALInteractiveRequestTests:testAuthorizationUri's sliceParameters dictionary becomes empty. If so, why do we have it at all? Should we just remove that? Seems an unnecessary layer of confusion at this point?